documentation/getting-started.adoc

```diff
@@ -0,0 +1,245 @@
+== Getting started
+
+=== Prerequisites
+
+Kubernetes or OpenShift cluster are required to deploy Strimzi. Strimzi supports all kind of clusters - from public and
+private clouds up to local deployments intended for development purposes. This guide expects that a Kubernetes or
```

Comment:
"up to" is wrong because the public or private clouds are large than/superior to a local deploymet. So it could be "down to" if you stick with local last, or you could keep "up to", but put local first in the sentence and put public and private clouds after the "up to".